other/upgrade_1-0.sql

```diff
@@ -1953,6 +1952,11 @@ VALUES
 DELETE FROM {$db_prefix}settings
 WHERE variable = 'agreement'
 LIMIT 1;
+
+REPLACE INTO {$db_prefix}settings
+	(variable, value)
+VALUES ('requireAgreement', '1'),
+	   ('requirePolicyAgreement', '1');
```

Comment:
The first edit to upgrade is fine since the option is being removed, but we shouldn't otherwise touch the upgrade.